.circleci/config.yml

```diff
@@ -138,38 +155,49 @@ jobs:
                --label "build.number=$CIRCLE_BUILD_NUM" \
                --label "build.url=$CIRCLE_BUILD_URL" \
                --label "build.gitref=$CIRCLE_SHA1"
+      - snyk/scan:
+          project: '${CIRCLE_PROJECT_REPONAME}-docker/${CIRCLE_BRANCH}'
+          docker-image-name: 'mojdigitalstudio/court-case-service:$APP_VERSION'
+          target-file: 'Dockerfile'
+          monitor-on-build: << parameters.master >>
+          <<: *snyk_options
+      - when:
+          condition: << parameters.master >>
+          steps:
+            - run:
+                name: Publish to docker hub
+                command: |
+                  docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+                  docker tag mojdigitalstudio/court-case-service:$APP_VERSION mojdigitalstudio/court-case-service:latest
+                  docker push mojdigitalstudio/court-case-service:$APP_VERSION
+                  docker push mojdigitalstudio/court-case-service:latest
+                  fi
```

Comment:
Looks strange to have this straggling "fi" closing a branch. IS that needed now you have the condition: above?

Reply:
good spot, this is a mistake

Reply:
fixed